mappings/net/minecraft/client/render/FirstPersonRenderer.mapping

```diff
@@ -42,6 +42,7 @@ CLASS none/box net/minecraft/client/render/FirstPersonRenderer
 		ARG 0 holder
 		ARG 1 stack
 		ARG 2 transformation
+		ARG 3 holdInLeftHand
```

Comment:
leftHanded here too? Not sure, you might've missed it or it might've been on purpose.